internal/config/profile.go

```diff
@@ -527,11 +550,20 @@ func (p *Profile) Rename(newProfileName string) error {
 	return nil
 }
 
-// Load loads the configuration from disk.
-func Load() error { return p.Load(true) }
-func (p *Profile) Load(readEnvironmentVars bool) error {
-	viper.SetConfigType(configType)
+func LoadAtlasCLIConfig() error { return Default().LoadAtlasCLIConfig(true) }
+func (p *Profile) LoadAtlasCLIConfig(readEnvironmentVars bool) error {
+	viper.SetConfigName("config")
+	return p.load(readEnvironmentVars)
+}
+
+func LoadMongoCLIConfig() error { return Default().LoadMongoCLIConfig(true) }
+func (p *Profile) LoadMongoCLIConfig(readEnvironmentVars bool) error {
 	viper.SetConfigName(ToolName)
+	return p.load(readEnvironmentVars)
+}
+
+func (p *Profile) load(readEnvironmentVars bool) error {
+	viper.SetConfigType(configType)
```

Comment:
just a reminder that `viper.SetEnvPrefix(EnvPrefix)` needs to change, not needed for this PR

Reply:
yes, I have this ticket https://jira.mongodb.org/browse/CLOUDP-112237 for that